### PR TITLE
Add support for ARM64 (Apple Silicon / M1)

### LINKED ISSUE #46: Plugin won't install on newer macs with Apple Silicon / M1 processor
When trying to install this plug in my Macbook air (with M1 processor) I get the following error:

```
kubectl krew install tree
Updated the local copy of plugin index.
Updated the local copy of plugin index "kvaps".
Installing plugin: tree
W0829 14:09:37.491565   12970 install.go:164] failed to install plugin "tree": plugin "tree" does not offer installation for this platform
F0829 14:09:37.491618   12970 root.go:79] failed to install some plugins: [tree]: plugin "tree" does not offer installation for this platform
```

Would it be possible to to add a `darwin/arm64` build target?

### PR DESCRIPTION
**Pull request by sammcj**
fixes #46

### CODE REVIEW

**Comment by sammcj** [comment]
Whoops missed that! , fixed 😄

File: .goreleaser.yml [hunk 1]
```diff
@@ -14,9 +14,6 @@ builds:
   goarch:
   - amd64
   - arm64
-  ignore:
```
Comment (ahmetb) [inline .goreleaser.yml:17]: test fails. don't remove this field?

Comment (sammcj) [reply .goreleaser.yml:17]: whoops! thanks

**Comment by sammcj** [comment]
🙇

**Pull request merged**